db/schema_0.sql

```diff
@@ -307,6 +307,23 @@ CREATE TABLE `target` (
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+
+--
+-- Table structure for table `oncall_team`
+--
+
+DROP TABLE IF EXISTS `oncall_team`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `oncall_team` (
+  `target_id` bigint(20) NOT NULL,
+  `name` varchar(255) NOT NULL,
```

Comment:
should name be unique? that field should at least have some index as it's being used in WHERE clauses